app/modules/sightings/schemas.py

```diff
@@ -58,6 +58,12 @@ class ElasticsearchSightingSchema(BaseSightingSchema):
 
     time = base_fields.Function(Sighting.get_time_isoformat_in_timezone)
     timeSpecificity = base_fields.Function(Sighting.get_time_specificity)
+    location = base_fields.Function(Sighting.get_locality)
```

Comment:
is this meant to be `locationId` or `verbatimLocality`?  or both?  maybe a question for @brmscheiner

Reply:
I understood that Ben wanted the real text from our chat, so what is verbatimLocality, although he referred to it as the locationId which is the guid.